tests/static_component/win32/simple.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+extern "C"
+{
+    void __stdcall SimpleFunction() noexcept;
```

Comment:
Just a guess, but I think the build failure is because Rust doesn't know how to handle multiple imports with the same name that have different link attributes (and thus come from different libs).

Reply:
Although they're compiled as separate tests so that's probably not the issue.

Reply:
Failing on release-compiled lib. Works fine with a debug-compiled lib. `rustc` is touching the lib in all cases so something _interesting™️_ is going on here. Investigating.